README.md

```diff
@@ -866,7 +908,24 @@ void listenWithConfig() {
 }    
 ```
 
+# Map Near Cache
+Map can be configured to work with near cache enabled. Near cache allows local caching of entries fetched from the cluster at the client side. The local cache is updated when the client does a get request for a map entry. The locally cached entries are updated automatically as any change occurs in the map data in the cluster. Special internal event listeners are utilized for this purpose.
+
+If you are doing mostly reads from the map rather than map updates, then enabling near-cache allows you to get the entries faster since it will return the locally cached entry which will eliminate any network interaction. But if your application is doing mostly writes (update, remove, etc.) from map, then you may not gain much by enabling the near-cache since the updates will cause network traffic. 
```

Comment:
..., then enabling near cache allows.......not gain much by enabling near cache since.....